components/BattleScreen/EnemyState/EnemyState.js

```diff
@@ -0,0 +1,99 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+const EnemyStateSection = styled.section`
+	position: absolute;
+	top: 20px;
+	left: 0px;
+	background-image: url("/sprites/statusbar-opponent3.png");
+	width: 200px;
+	height: 68px;
+	background-repeat: no-repeat;
+	background-size: contain;
+	z-index: 1;
+`;
+
+const EnemyPokemonName = styled.p`
+	position: absolute;
+	top: 5px;
+	left: 15px;
+	z-index: 1;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	text-transform: uppercase;
+	margin: 0;
+	padding: 0;
+`;
+
+const EnemyPokemonLevel = styled.span`
+	position: absolute;
+	top: 5px;
+	right: 29px;
+	z-index: 10;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+`;
+
+const EnemyPokemonLevelTitle = styled.span`
+	position: absolute;
+	top: 5px;
+	right: 50px;
+	z-index: 10;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+`;
+
+const EnemyPokemonHealthBarContainer = styled.progress`
```

Comment:
I didn't know that this HTML element exists, thank you!

Reply:
I did not know this morning yet, thanks internet! :-D It can be used well to display progress bars, for example, I have then converted to a HP display.